src/include/config-h.in.cmake

```diff
@@ -330,4 +330,7 @@
 /* Defined if getentropy() is available */
 #cmakedefine HAVE_GETENTROPY
 
+/* Defined if boost::context is available */
+#cmakedefine HAVE_BOOST_CONTEXT
```

Comment:
@cbodley is this macro used anywhere? or i am just missing something.

Reply:
sorry, nvm. i found it at https://github.com/ceph/ceph/pull/19054/files#diff-b5fd6691c22ab2c09a283b8bdda0e9ecR22